docs: Use the correct name case for microG (#54)
ReVanced/revanced-documentation: docs/revanced-resources/troubleshooting.md

## 15 App shows "No internet connection"

> This issue can be caused by recently changing your Google account password. Re-login to microG GmsCore, or reinstall it.

Reviewer: > on top of the existing installation

That is called an update, not a reinstall.

> This issue can be caused by recently changing your Google account password. Re-login to microG GmsCore, or reinstall it.

Reviewer: > But users may not know the difference. To them opening an APK may be the same as "installing" despite the dialog showing "install" vs "update".

Instead of "reinstall" you can also use the equivalent more explicit version "uninstall and install"

> This issue can be caused by recently changing your Google account password. Re-login into microG GmsCore, or uninstall then reinstall microG.

Reviewer: ```suggestion
This issue can be caused by recently changing your Google account password. Re-login to microG GmsCore, or re-install it.
```
